pkg/server/grpc.go

```diff
@@ -24,3 +29,40 @@ func (g *GRPC) Create(_ context.Context, req *customer.CreateRequest) (*customer
 
 	return &customer.Empty{}, nil
 }
+
+func NewGRPC(cfg *config.Cfg, store CustomerStore) (*GRPC, error) {
+	lis, err := net.Listen("tcp", cfg.Server.GRPC.Endpoint)
+	if err != nil {
+		return &GRPC{}, err
+	}
+
+	var opts []grpc.ServerOption
+	grpcServer := grpc.NewServer(opts...)
+
+	grpc := &GRPC{
+		logger:   cfg.Logger,
+		store:    store,
+		grpc:     grpcServer,
+		listener: lis,
+	}
+
+	customer.RegisterCustomerServer(grpcServer, grpc)
+
+	return grpc, nil
+}
+
+func (g *GRPC) Start(_ context.Context) error {
+	g.logger.Info("iniciando servidor gRPC")
+	err := g.grpc.Serve(g.listener)
```

Comment:
Dá pra retornar direto dessa linha:

```suggestion
	return g.grpc.Serve(g.listener)
```